scheduler/utilization_based_host_allocator.go

```diff
@@ -184,23 +188,26 @@ func evalHostUtilization(ctx context.Context, d distro.Distro, taskGroupData Tas
 		numNewHosts = 0
 	}
 
-	// alert if the distro is underwater
 	if maxHosts < 1 {
 		return 0, 0, errors.Errorf("unable to plan hosts for distro %s due to pool size of %d", d.Id, d.HostAllocatorSettings.MaximumHosts)
 	}
+	// Alert if we expect the total expected duration of all short running tasks to take over
+	// the dynamicDistroRuntimeAlertThreshold even when the distro's (or group's) max hosts
+	// are reached
 	underWaterAlert := message.Fields{
-		"provider":  d.Provider,
-		"distro":    d.Id,
-		"runtime":   scheduledDuration,
-		"runner":    RunnerName,
-		"message":   "distro underwater",
-		"num_hosts": len(existingHosts),
-		"max_hosts": d.HostAllocatorSettings.MaximumHosts,
-	}
-	avgMakespan := scheduledDuration / time.Duration(maxHosts)
+		"provider":        d.Provider,
+		"distro":          d.Id,
+		"runtime":         totalShortRunningTasksExpectedDuration,
+		"runner":          RunnerName,
+		"message":         "distro underwater",
```

Comment:
I'm concerned that we maybe aren't alerting on this? https://mongodb.splunkcloud.com/en-US/app/search/alerts Should we?

Reply:
The alert below it will go straight to the ops alerts channel ([example](https://mongodb.slack.com/archives/C866SR2LR/p1736371845288279))